ui/package.json

```diff
@@ -4,9 +4,9 @@
   "private": true,
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.7.2",
-    "@kubernetes/client-node": "scality/kubernetes-client-javascript.git#browser",
+    "@kubernetes/client-node": "git://github.com/scality/kubernetes-client-javascript.git#browser",
     "axios": "^0.18.0",
-    "core-ui": "scality/core-ui.git",
+    "core-ui": "github:scality/core-ui.git",
```

Comment:
Just wondering, should we use `github:<org>/<repo>` or `git://github.com/<org>/<repo>`? Since you did both, I guess both work?

Reply:
@NicolasT @carlito-scality Any opinion ?

Reply:
Yes, both works. we should use `github:<org>/<repo>` everywhere though. By the way, this change is already merged on development/2.0

Reply:
Let's just use the shortest one.